engine/src/main/scala/cromwell/server/WorkflowManagerSystem.scala

```diff
@@ -24,7 +24,10 @@ trait WorkflowManagerSystem {
   def allowedBackends: List[String] = ConfigFactory.load.getConfig("backend").getStringList("backendsAllowed").toList
   def defaultBackend: String = ConfigFactory.load.getConfig("backend").getString("defaultBackend")
 
+  val isServerMode = false
```

Comment:
I debated this but I don't have an opinion either way. It seemed simpler to just implement the val but I'm open to re-consider!